rhoas-cli-service-registry/README.adoc

```diff
@@ -97,6 +97,10 @@ This example creates a {registry} instance called `my-registry`.
 ----
 $ rhoas service-registry create --name my-registry
 Creating {registry} instance with name: my-registry
+----
```

Comment:
I don't seem to be able to tag the actual line, but on Line 90, delete the text “Enter the following command to”, and change “create” to “Create”.

Reply:
Done!